ofrak_core/ofrak/core/complex_block.py

```diff
@@ -176,3 +180,41 @@ async def analyze(
         :return: The virtual addresses of all the data words referenced by the complex block.
         """
         raise NotImplementedError()
+
+
+class ComplexBlockLinkableSymbolIdentifier(Identifier[None]):
+    """
+    ComplexBlocks are linkable, so tag them as LinkableSymbols.
+    """
+
+    targets = (ComplexBlock,)
```

Comment:
Better yet, per https://github.com/redballoonsecurity/ofrak/pull/115#discussion_r1027617094, I wonder if these analyzers are even needed...

Reply:
The Identifier and Analyzer structure in this PR is set up so that resources which can be linkable symbols are just tagged as that, and then analyzed when linkable symbols are needed. So that this call:
```
get_descendants_as_view(LinkableSymbol, r_filter=ResourceFilter.with_tags(LinkableSymbol))
```
work as expected. It gets all the resources tagged as LinkableSymbols (by the Identifiers) and then to get the view, analyzes them with the Analyzers.